samples/maps/geo-map/binding-multiple-shapes/Pages/MapBindingMultipleShapes.razor

```diff
@@ -4,16 +4,31 @@
 <div class="igContainer">
     <div class="igComponent">
         <GeographicMap Height="100%" Width="100%" Zoomable="true">
+            <GeographicShapeSeries ShapefileDataSource="@AsiaShape" />
+            <GeographicShapeSeries ShapefileDataSource="@EuropeShape" />
```

Comment:
can you set these props on series so shapes are more visible?
Outline="Black" Thickness="1" Brush="Red"
Outline="Black" Thickness="1" Brush="Purple"